helpers/content.go

```diff
@@ -173,11 +173,12 @@ func GetHTMLRenderer(defaultFlags int, ctx *RenderingContext) blackfriday.Render
 	}
 
 	return &HugoHtmlRenderer{
-		blackfriday.HtmlRendererWithParameters(htmlFlags, "", "", renderParameters),
+		Dir:      ctx.Dir,
```

Comment:
ooo, its not anymore! - I've an update coming, but i need to forward port it from the my v0.14 fork.